Package.swift

```diff
@@ -191,7 +191,9 @@ var streamChatSourcesExcluded: [String] { [
     "Workers/ChannelUpdater_Mock.swift",
     "Workers/EventNotificationCenter_Tests.swift",
     "Workers/ChatClientUpdater_Tests.swift",
+    "Query/FilterTestExtensions_Tests.swift",
```

Comment:
just to be sure, you did not edit this manually, right?

Reply:
Sure, runned script from Scripts

Reply:
You shouldn't run this script manually. Please run `./bootstrap.sh`, it'll install this script as a commit hook, so it runs automatically.